modules/tinymce/src/themes/silver/main/ts/ui/statusbar/ResizeHandle.ts

```diff
@@ -0,0 +1,80 @@
+import { AlloyEvents, Behaviour, Dragging, Focusing, NativeEvents, SimpleSpec, SimulatedEvent, Tabstopping } from '@ephox/alloy';
+import { Optional } from '@ephox/katamari';
+import { EventArgs, SugarPosition } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+import VK from 'tinymce/core/api/util/VK';
+
+import { UiFactoryBackstageProviders } from '../../backstage/Backstage';
+import { get as getIcon } from '../icons/Icons';
+import { resize, ResizeTypes } from '../sizing/Resize';
+
+const getResizeType = (editor: Editor): ResizeTypes => {
+  // If autoresize is enabled, disable resize
+  const fallback = !editor.hasPlugin('autoresize');
+  const resize = editor.getParam('resize', fallback);
+  if (resize === false) {
+    return ResizeTypes.None;
+  } else if (resize === 'both') {
+    return ResizeTypes.Both;
+  } else {
+    return ResizeTypes.Vertical;
+  }
+};
+
+const keyboardHandler = (editor: Editor, resizeType: ResizeTypes, event: SimulatedEvent<EventArgs<KeyboardEvent>>) => {
+  const scale = 3;
```

Comment:
- [ ] Find out if someone more qualified than me to make design decisions thinks that this is a good solution